src/OrleansServiceBus/Providers/Streams/EventHub/StatisticMonitors/DefaultEventHubBlockPoolMonitor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Providers.Streams.Common;
+using Orleans.Runtime;
+using Orleans.ServiceBus.Providers;
+
+namespace OrleansServiceBus.Providers.Streams.EventHub.StatisticMonitors
+{
+    /// <summary>
+    /// Default monitor for Object pool used by EventHubStreamProvider
+    /// </summary>
+    public class DefaultEventHubBlockPoolMonitor : DefaultBlockPoolMonitor
+    {
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="dimensions"></param>
+        /// <param name="logger"></param>
+        public DefaultEventHubBlockPoolMonitor(EventHubBlockPoolMonitorDimensions dimensions, Logger logger)
+            :base(logger)
+        {
+            this.LogProperties = new Dictionary<string, string>
+            {
+                {"Path", dimensions.EventHubPath},
+                {"ObjectPoolId", dimensions.BlockPoolId}
+            };
```

Comment:
Make the monitor aggregate different in DefaultEventHubBlockPoolMonitor , compared to DefaultBlockPoolMonitor . Since this is the more common case for event hub stream provider